src/components/feed/AddNode/AddNode.tsx

```diff
@@ -340,12 +354,21 @@ const AddNode: React.FC<AddNodeProps> = ({
 
   return (
     <React.Fragment>
-      <Button icon={<MdOutlineAddCircle />} type="button" onClick={toggleOpen}>
+      {readOnly ? (
+        <div className="tooltip-container">
+        <button className="tooltip-hoverable" disabled={true} >
+        {<MdOutlineAddCircle />} Add a Child Node{" "}( C )
+        </button>
+        <span className="tooltip">You are not the feed owner</span>
```

Comment:
Use patternfly's implementation of tooltip instead of a manual implementation.

https://www.patternfly.org/v4/components/button#aria-disabled-examples

Reply:
Done. Please review